tests/nested/core20/tpm/task.yaml

```diff
@@ -24,7 +12,8 @@ execute: |
     execute_remote "test -e /sys/kernel/security/tpm0/binary_bios_measurements"
 
     echo "and secure boot is enabled on the nested vm"
-    retry -n 5 --wait 5 execute_remote "test-snapd-mokutil --sb-state" | MATCH "SecureBoot enabled"
+    execute_remote "ls /sys/firmware/efi/efivars/SecureBoot-*"
```

Comment:
To verify if the secure boot is enabled, the attribute byte in this file should be set to 01. The attribute byte is the 5th byte in SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c.

Secure boot disabled:
```
0000000 06 00 00 00 00
0000005
```
Secure boot enabled:
```
0000000 06 00 00 00 01
0000005
```

Reply:
@cmatsuoka  thanks for the check, I'll update the test accordingly